pkg/internal/approver/allowed/validation.go

```diff
@@ -76,14 +76,32 @@ func (a allowed) Validate(_ context.Context, policy *policyapi.CertificateReques
 	}
 
 	for _, stringSlice := range stringSlices {
-		if stringSlice.slice != nil && stringSlice.slice.Required != nil && *stringSlice.slice.Required && stringSlice.slice.Values == nil {
-			el = append(el, field.Required(stringSlice.path.Child("values"), "values must be defined if required field"))
+		if stringSlice.slice != nil {
+			if stringSlice.slice.Required != nil && *stringSlice.slice.Required {
+				if stringSlice.slice.Values == nil && len(stringSlice.slice.Validations) == 0 {
+					el = append(el, field.Required(stringSlice.path.Child("values"), "values/validations must be defined if required field"))
```

Comment:
```suggestion
					el = append(el, field.Required(stringSlice.path.Child("values"), "values/validations must be defined if field is required"))
```
? or is this the "don't show this error to the user, it isn't human-readable" bit?

Reply:
No, this is a user-facing error. 😉 I think the suggestion from @inteon is even better.